examples/generate-pipeline/README.adoc

```diff
@@ -20,22 +20,8 @@ kubectl create clusterrolebinding serviceaccounts-cluster-admin \
 To generate and run a pipeline:
 
 * Run skaffold generate-pipeline
-* Modify skaffold.yaml to use a valid GCSbucket for kaniko if needed.
+* Modify skaffold.yaml to use a valid GCSbucket for kaniko
 * Commit and push updated skaffold.yaml
 * kubectl apply -f pipeline.yaml
-* Create a pipelinerun.yaml 
-----
-apiVersion: tekton.dev/v1alpha1
-kind: PipelineRun
-metadata:
-  name: skaffold-pipeline-run
-spec: 
-  pipelineRef: 
-    name: skaffold-pipeline
-  resources:
-  - name: source-repo
-    resourceRef:
-      name: source-git
-----
- 
+* Create a pipelinerun.yaml
```

Comment:
Are these intentional?

Reply:
yes, these are copied by the hack/release.sh script